### DISCUSSION TITLE
How to pass --cfc within the script?

### DISCUSSION THREAD

**Discussion by peterc2609** [usnistgov/macos_security#395] [Q&A] [answered]
I want to run these scripts from Intune silently. So they are non interactive.

With Intune, I'm unable to pass flags to a shell script, so I need to modify the script so that it thinks --cfc has been passed on the command line, but really its within the script i.e. it will always run as --cfc...

Whats the best way to do it?

**Comment by jamierrichardson** [comment, marked as answer]
See answer in #394
